src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1578,6 +1572,10 @@ format layout_optimizer::get_preferred_format(program_node& node) {
         if (only_gemm_users(node)) {
             // TODO: Gemm is not supporting fsv layouts
             expected = format::bfyx;
+        } else if (use_onednn_impls && (layout.data_type == data_types::i8 || layout.data_type == data_types::u8) &&
+            needs_all_usr_onednn_bfyx_to_blocked(format::bfyx, node)) {
+            // All user nodes are convolutions which satisfy options for onednn first conv
+            expected = format::bs_fs_yx_bsv8_fsv4;
```

Comment:
I think this condition should be fused into existing condition. For example the one on line 1586. That is the previous condition for first conv..